scripts/start/projects/project_WetHorizons.R

```diff
@@ -0,0 +1,132 @@
+# |  (C) 2008-2025 Potsdam Institute for Climate Impact Research (PIK)
+# |  authors, and contributors see CITATION.cff file. This file is part
+# |  of MAgPIE and licensed under AGPL-3.0-or-later. Under Section 7 of
+# |  AGPL-3.0, you are granted additional permissions described in the
+# |  MAgPIE License Exception, version 1.0 (see LICENSE file).
+# |  Contact: magpie@pik-potsdam.de
+
+# ----------------------------------------------------------
+# description: Project WetHorizons Peatland
+# position: 5
+# ----------------------------------------------------------
+
+## Load lucode2 and gms to use setScenario later
+library(lucode2)
+library(gms)
+library(gdx2)
+library(magpie4)
+
+# Load start_run(cfg) function which is needed to start MAgPIE runs
+source("scripts/start_functions.R")
+
+# Source default cfg. This loads the object "cfg" in R environment
+source("config/default.cfg")
+
+# create additional information to describe the runs
+cfg$info$flag <- "WH01"
+
+cfg$results_folder <- "output/:title:"
+cfg$results_folder_highres <- "output"
+cfg$force_replace <- TRUE
+cfg$force_download <- FALSE
+cfg$qos <- "standby_dayMax"
+
+# support function to create standardized title
+.title <- function(cfg, ...)
+  return(paste(cfg$info$flag, sep = "_", ...))
+
+cfg$repositories <- append(
+  list(
+    "https://rse.pik-potsdam.de/data/magpie/public" = NULL,
+    "./patch_inputdata" = NULL
+  ),
+  getOption("magpie_repos")
+)
+
+cfg$gms$c_timesteps <- "5year"
+ssp <- "SSP2"
+
+# Reference
+cfg$title <- .title(cfg, paste(ssp, "Ref", sep = "-"))
+cfg <- setScenario(cfg, c(ssp, "NPI", "rcp7p0"))
+cfg$gms$c56_mute_ghgprices_until <- "y2150"
+cfg$gms$c56_pollutant_prices <- paste0("R32M46-", if (ssp == "SSP2") "SSP2EU" else ssp, "-NPi")
```

Comment:
I thought SSP2EU was removed? Can we simplify like this then?
```suggestion
cfg$gms$c56_pollutant_prices <- paste0("R32M46-", ssp, "-NPi")
```

Reply:
Ah it's probably still present in this older remind/magpie version that is addressed here, right?

Reply:
This is copy-paste from the test_runs.R script.
`SSP2EU` is included in the name of the GHG price trajectory. 
In this specific case I can just change it to the actual name.